Project.toml

```diff
@@ -4,37 +4,23 @@ authors = ["David Lung <lungdm@gmail.com> and contributors"]
 version = "0.1.0"
 
 [deps]
-BenchmarkTools = "6e4b80f9-dd63-53aa-95a3-0cdb28fa8baf"
-BlackBoxOptim = "a134a8b2-14d6-55f6-9291-3336d3ab0209"
-CMAEvolutionStrategy = "8d3b24bd-414e-49e0-94fb-163cc3a3e411"
-Cthulhu = "f68482b8-f384-11e8-15f7-abe071a5a75f"
 DiffEqBase = "2b5f629d-d688-5b77-993f-72d75c75574e"
 DiffEqCallbacks = "459566f4-90b8-5000-8ac3-15dfb0a30def"
 DiffEqFlux = "aae7a2af-3d4f-5e19-a356-7da93b79d9d0"
 DiffEqSensitivity = "41bf760c-e81c-5289-8e54-58b1f1f8abe2"
-DifferentialEquations = "0c46a032-eb83-5123-abaf-570d42b7fbaa"
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 Evolutionary = "86b6b26d-c046-49b6-aa0b-5f0f74682bd6"
 Flux = "587475ba-b771-5e3f-ad9e-33799f191a9c"
-GR = "28b8d3ca-fb5f-59d9-8090-bfdbd6d07a71"
 GalacticOptim = "a75be94c-b780-496d-a8a9-0878b188d577"
 IterTools = "c8e1da08-722c-5040-9ed9-7db0dc04731e"
-Juno = "e5e0dc1b-0480-54bc-9374-aad01c23163d"
 MLDataPattern = "9920b226-0b2a-5f5f-9153-9aa70a013f8b"
 MLDataUtils = "cc2ba9b6-d476-5e6d-8eaf-a92d5412d41d"
 ModelingToolkit = "961ee093-0014-501f-94e3-6117800e7a78"
 NLopt = "76087f3c-5699-56af-9a33-bf431cd00edd"
-NPZ = "15e1cf62-19b3-5cfa-8e77-841668bca605"
```

Comment:
NPZ is needed to load the half-cheetah data in https://github.com/lungd/LTC.jl/blob/master/example/half_cheetah_data_loader.jl#L10.
We should probably store data as txt/csv

Reply:
We should add test dependencies separately. I'll do that shortly.

Reply:
Since the repo doesn't have any tests yet I think let's merge this first and move forward from a different PR